functests/10-test-conditions-templates.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+[ -z ${KV_NAMESPACE} ] && exit 127
+
+RET=1
+TEST_NS="${KV_NAMESPACE}"
+
+oc create -n ${KV_NAMESPACE} -f "${SCRIPTPATH}/common-templates-versioned-cr.yaml" || exit 2
+
+# 76 is number of all templates in version v0.6.0
```

Comment:
I am not sure we should hardcode this number. I will require maintenance and is pretty well hidden. Is there any other way?

Reply:
I don't like it either. I can try take version from CR and then e.g. count how many templates are in the file.

Reply:
What about reading the ansible run status from the CR itself? If the ansible finished, then all templates must surely be there.

Reply:
good idea